### PR TITLE
Remove legacy captains and add Michael Irwin

### PR DESCRIPTION
**Pull request by mikesir87**
Signed-of-by: Michael Irwin <mikesir87@gmail.com>

### CODE REVIEW

**Comment by mikesir87** [comment]
Should we keep a separate list of "legacy/retired" captains so folks can still find their Twitter accounts/blogs? Thoughts?

File: content/en/docs/captains/twitter/_index.md [hunk 1]
```diff
@@ -23,13 +22,11 @@ description: >-
   - **Nick Janetakis** - [@nickjanetakis](https://twitter.com/nickjanetakis) - [Blog](https://nickjanetakis.com) - [YouTube](https://www.youtube.com/c/nickjanetakis)
   - **Nils De Moor** - [@ndemoor](https://twitter.com/ndemoor) - [Blog](https://cntnr.io)
   - **Nigel Poulton** - [@nigelpoulton](https://twitter.com/nigelpoulton) - [Blog](http://nigelpoulton.com)
-  - **Phil Estes** - [@estesp](https://twitter.com/estesp) - [Blog](https://integratedcode.us)
   - **Philipp Garbe** - [@pgarbe](https://twitter.com/pgarbe) - [Blog](http://garbe.io)
   - **Shawn Bower** - [@drizzt51](https://twitter.com/drizzt51) - [Blog](https://medium.com/@shawn.bower)
-  - **Stefan Scherer** - [@stefscherer](https://twitter.com/stefscherer) - [Blog](https://stefanscherer.github.io)
   - **Tim Tyler** - [@timotyler](https://twitter.com/timotyler) - [Blog](https://medium.com/@tim.tyler)
   - **Adrian Mouat** - [@adrianmouat](https://twitter.com/adrianmouat) - [Blog](https://container-solutions.com/author/adrianm/)
-  - **Nicolas De Loof** - [@ndeloof](https://twitter.com/ndeloof) - [Blog](http://blog.loof.fr)
```
Comment (ndeloof) [inline content/en/docs/captains/twitter/_index.md:32]: :'(

**Comment by ajeetraina** [comment]
> Should we keep a separate list of "legacy/retired" captains so folks can still find their Twitter accounts/blogs? Thoughts?

That's a good idea. Let me check if we can come up with separate child pages around it.

**Review by ajeetraina — APPROVED** [review]

File: content/en/docs/captains/twitter/_index.md (hunk 1 above)
Comment (mikesir87) [reply content/en/docs/captains/twitter/_index.md:32]: Haha... I know it! But hey! You're not lost forever... we still get to hang out! 😄

**Pull request merged**